packages/contracts/contracts/libraries/trie/Lib_MerkleTrie.sol

```diff
@@ -78,35 +78,6 @@ library Lib_MerkleTrie {
         return (exists && Lib_BytesUtils.equal(_value, value));
     }
 
-    /**
-     * @notice Updates a Merkle trie and returns a new root hash.
-     * @param _key Key of the node to update, as a hex string.
-     * @param _value Value of the node to update, as a hex string.
-     * @param _proof Merkle trie inclusion proof for the node *nearest* the
-     * target node. If the key exists, we can simply update the value.
-     * Otherwise, we need to modify the trie to handle the new k/v pair.
-     * @param _root Known root of the Merkle trie. Used to verify that the
-     * included proof is correctly constructed.
-     * @return _updatedRoot Root hash of the newly constructed trie.
-     */
-    function update(
-        bytes memory _key,
-        bytes memory _value,
-        bytes memory _proof,
-        bytes32 _root
-    ) internal pure returns (bytes32 _updatedRoot) {
-        // Special case when inserting the very first node.
-        if (_root == KECCAK256_RLP_NULL_BYTES) {
-            return getSingleNodeRootHash(_key, _value);
-        }
-
-        TrieNode[] memory proof = _parseProof(_proof);
-        (uint256 pathLength, bytes memory keyRemainder, ) = _walkNodePath(proof, _key, _root);
-        TrieNode[] memory newPath = _getNewPath(proof, pathLength, _key, keyRemainder, _value);
-
-        return _getUpdatedTrieRoot(newPath, _key);
```

Comment:
You could also delete this function's implementation (and any subsequent functions called only by it).

Reply:
done